### ISSUE TITLE
Cannot paste characters like ñ on os x

### ISSUE THREAD

**Issue by asweigart** [asweigart/pyperclip#60]
This bug report comes from this user comment:

In Lecture 29, I have a large PDF directory with phone numbers and emails. I have tested the script and it works on one page with phone numbers and emails, but when I do the whole PDF (e.g. Ctrl-A, then Ctrl-C), I get the following error:

File "phone_email_ex.py", line 33, in <module>

text = pyperclip.paste()

File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/pyperclip/clipboards.py", line 22, in paste_osx

return stdout.decode('utf-8')

UnicodeDecodeError: 'utf-8' codec can't decode byte 0xd2 in position 23578: invalid continuation byte

I think it is an issue of encoding, but where do I find the issue in the data (has to be some random character like an "ñ") and where do I put the appropriate encoding after importing the encodings module. It looks like it should be during the pyperclip.paste() function.

**Comment by contrixed** [comment]
Why not use the solution in #18?

**Comment by jaraco** [comment]
Yep. I've been using [jaraco.clipboard](https://pypi.io/project/jaraco.clipboard) with success even with non-ascii characters for some time. I highly recommend it.

**Comment by hgiroto** [comment]
Estimado maestro Al, usa el módulo XEROX!!! Acepta hasta mismo caracteres como ñ, ç, ó... etc...
Además, acepta para el INPUT que el texto copiado al clipboard tenga \n (<ENTER>).
